cloud-sql/mysql/mysql/server.js

```diff
@@ -83,19 +83,24 @@ const pool = mysql.createPool({
   // [END cloud_sql_mysql_mysql_backoff]
 
   //[END_EXCLUDE]
-});
+};
+let pool;
+(async () => {
+  try {
+    pool = await mysql.createPool(connectionOptions);
+    // Wait for tables to be created (if they don't already exist).
+    await pool.query(
```

Comment:
Can we move this (database init) into it's own function? I don't make sense in the context of the snippet to include a query.